coverage/coverage.json

```diff
@@ -0,0 +1 @@
":123,"column":18}},"51":{"start":{"line":118,"column":19},"end":{"line":122,"column":20}},"52":{"start":{"line":119,"column":23},"end":{"line":119,"column":87}},"53":{"start":{"line":121,"column":23},"end":{"line":121,"column":72}},"54":{"start":{"line":129,"column":0},"end":{"line":129,"column":24}}},"branchMap":{"1":{"line":27,"type":"if","locations":[{"start":{"line":27,"column":8},"end":{"line":27,"column":8}},{"start":{"line":27,"column":8},"end":{"line":27,"column":8}}]},"2":{"line":39,"type":"if","locations":[{"start":{"line":39,"column":8},"end":{"line":39,"column":8}},{"start":{"line":39,"column":8},"end":{"line":39,"column":8}}]},"3":{"line":50,"type":"if","locations":[{"start":{"line":50,"column":8},"end":{"line":50,"column":8}},{"start":{"line":50,"column":8},"end":{"line":50,"column":8}}]},"4":{"line":75,"type":"if","locations":[{"start":{"line":75,"column":8},"end":{"line":75,"column":8}},{"start":{"line":75,"column":8},"end":{"line":75,"column":8}}]},"5":{"line":99,"type":"if","locations":[{"start":{"line":99,"column":8},"end":{"line":99,"column":8}},{"start":{"line":99,"column":8},"end":{"line":99,"column":8}}]},"6":{"line":114,"type":"if","locations":[{"start":{"line":114,"column":7},"end":{"line":114,"column":7}},{"start":{"line":114,"column":7},"end":{"line":114,"column":7}}]},"7":{"line":118,"type":"if","locations":[{"start":{"line":118,"column":19},"end":{"line":118,"column":19}},{"start":{"line":118,"column":19},"end":{"line":118,"column":19}}]}}},"/home/sven/pse/tageler-api/src/models/tageler.js":{"path":"/home/sven/pse/tageler-api/src/models/tageler.js","s":{"1":1,"2":1,"3":1,"4":1,"5":1,"6":0,"7":1,"8":0,"9":1,"10":0,"11":1,"12":0},"b":{},"f":{"1":0,"2":0,"3":0,"4":0},"fnMap":{"1":{"name":"(anonymous_1)","line":66,"loc":{"start":{"line":66,"column":36},"end":{"line":66,"column":61}}},"2":{"name":"(anonymous_2)","line":71,"loc":{"start":{"line":71,"column":28},"end":{"line":71,"column":58}}},"3":{"name":"(anonymous_3)","line":75,"loc":{"start":{"line":75,"column":32},"end":{"line":75,"column":55}}},"4":{"name":"(anonymous_4)","line":79,"loc":{"start":{"line":79,"column":29},"end":{"line":79,"column":47}}}},"statementMap":{"1":{"start":{"line":1,"column":0},"end":{"line":1,"column":37}},"2":{"start":{"line":2,"column":0},"end":{"line":2,"column":45}},"3":{"start":{"line":5,"column":0},"end":{"line":62,"column":3}},"4":{"start":{"line":64,"column":0},"end":{"line":64,"column":74}},"5":{"start":{"line":66,"column":0},"end":{"line":68,"column":2}},"6":{"start":{"line":67,"column":4},"end":{"line":67,"column":43}},"7":{"start":{"line":71,"column":0},"end":{"line":73,"column":2}},"8":{"start":{"line":72,"column":12},"end":{"line":72,"column":38}},"9":{"start":{"line":75,"column":0},"end":{"line":77,"column":2}},"10":{"start":{"line":76,"column":4},"end":{"line":76,"column":42}},"11":{"start":{"line":79,"column":0},"end":{"line":81,"column":2}},"12":{"start":{"line":80,"column":2},"end":{"line":80,"column":25}}},"branchMap":{}}}
```

Comment:
Please get rid of the rest of the `coverage` folder as well. All of it is regenerated as needed.